### ISSUE TITLE
inputs should have labels

### ISSUE THREAD

**Issue by aric87** [nikthequik/clashnode#3]
https://github.com/nikthequik/clashnode/blob/master/index.html#L9

Even if they're hidden, inputs should have a label for screen readers to tell users what the input is for
